contracts/Gate.sol

```diff
@@ -214,92 +279,112 @@ contract Gate is ProtocolVersioned, Owned {
         return (amountUT, nonce, unlockHeight, stakingIntentHash);
     }
 
+    /**
+      * @notice external function to process staking.
+      *
+      * @dev Bounty amount is to msg.sender if its not whitelisted worker
```

Comment:
Please add a word like "transferred" or "sent" between "amount is" and "to" in both lines here.

"its" --> "it's"